base_local_planner/include/base_local_planner/speed_limiters/speed_limiter.h

```diff
@@ -69,6 +69,7 @@ class SpeedLimiter {
    */
   virtual bool calculateLimits(double& max_allowed_linear_vel, double& max_allowed_angular_vel) = 0;
 
+  std::string string_name;
```

Comment:
If you make a member variable, use a trailing_underscore_ on the name.  That said, I'd make the name a required virtual function and then the child classes can implement it and return their string name.